draft-ietf-quic-transport.md

```diff
@@ -3251,12 +3251,17 @@ using for private experimentation on the GitHub wiki at
 In order to avoid ossification of the version number defined by this draft,
 servers announce a list of version numbers that they interpret as aliases for
 the version number used in this draft. Alias versions MUST NOT be a reserved
-version. Servers SHOULD send at least one version alias, and SHOULD frequently
-change the value that they announce. Each version alias contains a lifetime,
-which indicates how long the server will accept this version alias. It also
-contains an initial salt, which is used instead of the initial salt as defined
-in section 5.2 of {{QUIC-TLS}}. The list of version aliases is sent in the
-server's Transport Parameters (see {{transport-parameter-definitions}}).
+version. A server MUST NOT advertise an alias version number for a version that
+it actually supports. If the server advertises an alias version number that the
+client actually supports, the client MUST assume the server doesn't support
+that version and ignore the alias.
```

Comment:
I don't know that the client has to ignore the alias, per se -- it just needs to be cognizant of what the interpretation of that version will be on that server.  On the other hand, if there's a possibility that the server might acquire support for that version between flights (server upgrade case), perhaps it's better that the client just avoids it entirely after all.